assets/dev/scss/editor/panel/controls/_media.scss

```diff
@@ -130,13 +133,14 @@
 		width: 20px;
 		height: 20px;
 		font-size: 11px;
-		color: $editor-light;
-		background-color: rgba(#fff, .9);
+		color: #fff;
+		background-color: rgba($editor-dark, .85);
 		border-radius: 3px;
 		box-shadow: 0 0 7px 0 rgba(0, 0, 0, 0.1);
+		transition: background .3s;
 
 		&:hover {
-			background-color: rgba(#fff, 1);
+			background-color: rgba($editor-dark, .95);
```

Comment:
how are all those color changes related to the aspect ratio change?
should be in a different PR